wrappers/kn/README.md

```diff
@@ -0,0 +1,40 @@
+# ZXing-C++ Kotlin/Native Library
+
+## Install
+
+The easiest way to use the library is to fetch if from _mavenCentral_. Simply add
+
+```gradle
+implementation("io.github.zxing-cpp:kotlin-native:2.2.1")
+```
+
+to your `build.gradle.kts` file in the `dependencies` section of `nativeMain` source set.
+
+## Use
+
+A trivial use case looks like this:
+
+```kotlin
+import zxingcpp.BarcodeReader
+import zxingcpp.ImageView
+
+val options = ReaderOptions()
```

Comment:
Both of them will work, but the latter one is equal to following code:
```
val o = ReaderOptions()
o.formats = setOf(EAN13, QRCode)
o.tryHarder = true
o.maxNumberOfSymbols = 3
```
I can say which one is more "native", they achieve same thing in just different way, but If you ask me to choose one from them, I like the first one(awa keep both of them)

Reply:
There seems to be a confusion: the "first" one would be the `ReaderOptions.apply {...}` one. I prefer that one because of the symmetry to the other wrappers that are based on the c-api and the fact that the defaults don't need to be kept in sync. With "first" you seem to mean the constructor one?

Anyway, as you agree that it is only a question of style basically, I'd like to do it without the constructor parameters, which also results in less code.

Reply:
Also there is a bit of a performance argument to be made, since the constructor version will always call all setter functions, while the other only those that are actually used. Admittedly not relevant in practice, just saying ;).

Reply:
I your comment about derive BarcodeReader from ReaderOptions. If we adapt that, we'll have not choice on this question, users will have to use codes like following:
```
val reader = BarcodeReader().apply {
    formats = setOf(EAN13, QRCode)
    tryHarder = true
    maxNumberOfSymbols = 3
    // more options, see documentation
}
```
So I will mark this as resolved. :)